applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element_volume_error_source.cpp

```diff
@@ -0,0 +1,1836 @@
+			vel_original(2,0) = 0.2; vel_original(2,1) = 0.3; vel_original(2,2) = 0.4;
+			vel_original(3,0) = 0.3; vel_original(3,1) = 0.4; vel_original(3,2) = 0.5;
+
+			// Set the nodal BODY_FORCE, DENSITY and DYNAMIC_VISCOSITY values
+			for (NodeIteratorType it_node=modelPart.NodesBegin(); it_node<modelPart.NodesEnd(); ++it_node){
+				it_node->FastGetSolutionStepValue(DENSITY) = pElemProp->GetValue(DENSITY);
+				it_node->FastGetSolutionStepValue(DYNAMIC_VISCOSITY) = pElemProp->GetValue(DYNAMIC_VISCOSITY);
+				it_node->FastGetSolutionStepValue(BODY_FORCE_Z) = -9.81;
+			}
+
+			for(unsigned int i=0; i<4; i++){
+				pElement->GetGeometry()[i].FastGetSolutionStepValue(PRESSURE)    = 0.0;
+				for(unsigned int k=0; k<3; k++){
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY)[k]    = vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 1)[k] = 0.9*vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 2)[k] = 0.75*vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY)[k]    = 0.0;
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 1)[k] = 0.0;
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 2)[k] = 0.0;
+				}
+			}
+			pElement->GetGeometry()[0].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[1].FastGetSolutionStepValue(DISTANCE) =  1.0;
+			pElement->GetGeometry()[2].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[3].FastGetSolutionStepValue(DISTANCE) =  1.0;
+
+			// Compute RHS and LHS
+			Vector RHS = ZeroVector(16);
+			Matrix LHS = ZeroMatrix(16,16);
+
+			const auto& r_process_info = modelPart.GetProcessInfo();
+			pElement->Initialize(r_process_info); // Initialize the element to initialize the constitutive law
+			pElement->CalculateLocalSystem(LHS, RHS, r_process_info);
+			// Check the RHS values (the RHS is computed as the LHS x previous_solution,
+			// hence, it is assumed that if the RHS is correct, the LHS is correct as well)
+			std::cout << std::setprecision(12) << std::endl;
+        	
+			KRATOS_CHECK_NEAR(RHS(0), 3782.5533335, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(1), 2744.97890633, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(2), 3686.9679704, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(3), -5.52594881831, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(4), -3777.85819668, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(5), 237.470888418, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(6), -453.676393258, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(7), -4.00642325692, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(8), 10.2704695093, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(9), -3395.0486301, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(10), -469.951076124, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(11), -2.86019359732, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(12), 30.7635603322, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(13), 507.703002014, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(14), -4253.86133435, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(15), -4.37410099412, 1e-7);

```

Comment:
`KRATOS_CHECK_VECTOR_NEAR`.

Reply:
Removed this test and added to a existing file, using KRATOS_CHECK_VECTOR_NEAR. Done!